shared/src/vyxal/Elements.scala

```diff
@@ -1515,6 +1513,38 @@ object Elements:
       case a: VNum => a ** 3
       case a: String => VList.from(a.grouped(3).toSeq)
     },
+    addPart(
+      Monad,
+      "ɾ",
+      "Inclusive One Range | Uppercase",
+      List("1..n", "one->n", "one-range", "to-upper", "upper", "uppercase"),
```

Comment:
I didn't think `.` was valid in literate mode keywords.

Reply:
Umm... well it didn't give me any errors :p

Reply:
Well of course not

There's not tests or checks for that

Reply:
Fine. I'll change it